lib/Gemfile

```diff
@@ -78,7 +78,7 @@ unless ENV['APPLIANCE']
     gem "jasmine",    "~>2.0",    :require => false
```

Comment:
Note, line 75 above has `unless ENV['APPLIANCE']` so these test gems shouldn't be included in appliances.
@JPrause @jvlcek I'm merging this without your ack, just yell if this doesn't make sense.

thanks!